### PR TITLE
Comma separated list of animations

### PR DESCRIPTION
**Pull request by DarKDinDoN**
Hey,

Related to your feature suggestion: https://github.com/otigon/automated-jb2a-animations/pull/219#issuecomment-917633862

Here is the implementation. Hope I saved you some time since I wasted a bit of your time on permissive recognition.

### CODE REVIEW

**Comment by otigon** [comment]
I'll take a look at this this week. I know I was probably a bit brash but it was a little off putting that I spent the better part of a month getting this release together and people already want to change it.

That being said, I'll take a look at the PR this and see how it fits

**Comment by DarKDinDoN** [comment]
@otigon Hey, no prob.

Generally speaking, I know how it feels to have someone suggesting/creating a better/new/different implementation/interface/feature after spending a lot of time on a release.
It can be very frustrating not to have people just being simply happy with the current state of a module development. But remember, people just want the best out of an already great module!

That being said, don't let ego get in the way of a better module.
Good ideas that fit within the scope of the module should always be welcomed, either directly implemented or planned for future releases.
And if you don't have the time or the resources, just ask the community. We can help. We are thankful for your work. I know I am.

This is the way :)

**Comment by DarKDinDoN** [comment]
Hey,

I think we can close this, no?
Or if you want this to be implemented, we can still add it, but I think it requires a new PR. I can do that if you want :)

**Comment by DarKDinDoN** [comment]
Yes, don't have time now, but we need to be careful.
Since the last PR, names are now sorted from longest to shortest.

With the comma delimited list, the result of the sort would be:
- (0) sword, fire sword, sword of fire, sword of ice
- (1) Greatsword

Right now, with an item named "Greatsword", "(0) sword" would be a match for Greatsword.
So it means that we need to split the animation names before the array/object of default animation is sorted.

And to be honest, I'm a bit worried about performances if a user as like a 100 of default animations with a lot of "comma" in there.

I'm thinking that we can actually process the default animations array when a user saves the configuration window and store it in a hidden setting. So we don't have to process this for each and every animation.

Long story short, might not be that easy since the last PR :)

**Comment by otigon** [comment]
yeah you're probably right on that. I think the includes() method captures the most bang for your buck. Reskinned items can just have a new automatic detection added since those should be few and far between

**Comment by otigon** [comment]
Going to go ahead and close this out
